README.md

```diff
@@ -1 +1,90 @@
-# celery-logger
+# Celery Logger
+
+celery-logger is a python library for logging [celery](https://docs.celeryproject.org/en/stable/) events such as tasks received, tasks failed/succeeded and tasks retried, along with the passed args.
+
+## Features
+
+- Simple and flexible task logs
+- Multiple deployment options (docker, virtual machines)
+- Integration possibilities: ELK stack and aws cloudwatch for example
```

Comment:
references to aws in technical docs, is often written in UPPERCASE

Reply:
thank you sir!